tests/test_other.py

```diff
@@ -11963,3 +11963,14 @@ def test_no_cfi(self):
   @also_with_wasm_bigint
   def test_parseTools(self):
     self.do_other_test('test_parseTools.c', emcc_args=['--js-library', test_file('other/test_parseTools.js')])
+
+  def test_lto_atexit(self):
+    self.emcc_args.append('-flto')
```

Comment:
Do we also need optimizations here, or is just LTO enough to show the issue?

Reply:
LTO is enough.  I verified this test fails without this change.